lib/codelog/cli.rb

```diff
@@ -4,6 +4,13 @@
 
 module Codelog
   class CLI < Thor
+    map ['--version', '-v'] => :__print_version
+
+    desc '[--version, -v]', 'prints current codelog version'
+    def __print_version
```

Comment:
@emilianoLeite I guess the double underlines on the method name here were used so the command `codelog version` does not conflict with `codelog --version`. 
Are we planning to eventually implement a `codelog version` command for something else? 
I think it would make it quite confusing having two different action for very similar commands. Besides the printed text makes it clear that the version number refers to the gem, not the the changelog.
I don't see a problem in the gem version being accesible by either `codelog --version` or `codelog version`. What do you think?